infra/aws/terraform/artifacts.k8s.io/s3/variables.tf

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+variable "prefix" {
+  type    = string
+  default = ""
+}
+
+variable "s3_replication_iam_role_arn" {
+  description = "IAM role assumed by S3 service for replication"
+  type        = string
+  default     = ""
```

Comment:
It doesn't make sense to default to `""`. Either default to `null` and make replication condition, or set this to `nullable = false`, _don't_ set a default, and require the caller to set it.

Reply:
Thanks - it's triggered by s3_replication_rules, so it's not required.  I'll default it to null